src/common/IconButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+IconButton.propTypes = {
+  imgUrl: PropTypes.string.isRequired,
+  altText: PropTypes.string.isRequired,
+  onClick: PropTypes.func,
+}
+
+export default function IconButton({ imgUrl, altText, onClick }) {
+  return (
+    <ButtonContainer onClick={onClick}>
+      <img src={imgUrl} alt={altText} />
+    </ButtonContainer>
+  )
+}
+
+const ButtonContainer = styled.button`
+  width: 40px;
+  height: 40px;
```

Comment:
Should your IconButton have the 40x40 size on every display size? Otherwise I would use a more responsive size, like you did at other parts of your code.

Reply:
Yes, the size is fixed for every display size. I don't think it makes sense to have huge Icons on Desktop screens.